packages/components/pagination/src/pagination.story.js

```diff
@@ -36,21 +36,27 @@ storiesOf('Components|Pagination', module)
       />
     </Section>
   ))
-  .add('PageSizeSelector', () => (
-    <Section>
-      <Value
-        defaultValue={20}
-        render={(pageSize, onPageSizeChange) => (
-          <PageSizeSelector
-            pageSize={pageSize}
-            pageSizeRange={'s'}
-            currentPageItems={1}
-            onPageSizeChange={onPageSizeChange}
-          />
-        )}
-      />
-    </Section>
-  ))
+  .add('PageSizeSelector', () => {
+    const range = select('range', ['s', 'm', 'l'], 's');
```

Comment:
The `s` and `m` will have the same `20` value. Is that on purpose

Reply:
yes, since `20` is the minimum pageSize in respective range